lib/cursor_test.go

```diff
@@ -5,11 +5,12 @@ import (
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
+	"github.com/qri-io/qri/base/params"
 )
 
 func TestCursorToParams(t *testing.T) {
-	lp := ListParams{
-		OrderBy: "created",
+	lp := params.List{
```

Comment:
Question about `Cursor`: should the `ToParams` method add the `params.List` information into the `nextPage.url` string?